app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -27,9 +27,10 @@ import org.mozilla.tv.firefox.widget.InlineAutocompleteEditText
 
 class ScreenController {
 
-    private var _currentActiveScreen = MutableLiveData<ActiveScreen>().apply {
+    private val _currentActiveScreen = MutableLiveData<ActiveScreen>().apply {
         value = ActiveScreen.NAVIGATION_OVERLAY
     }
+    // This is updated just before the fragment transaction is committed
```

Comment:
nit: use property kdoc rather than "//": that indicates the comment is for consumers rather than file private comments.

Also, the comment should be more explicit, "Observers will be notified just before the fragment transaction is committed."